data/json/items/gun/762.json

```diff
@@ -30,6 +30,38 @@
     "flags": ["MODE_BURST", "NEVER_JAMS"]
   },
   {
+    "id": "arx160",
+    "type": "GUN",
+    "symbol": "(",
```

Comment:
Yeah, the magazine thing (and possessing a Cutting field) was a tip-off that it had been missed on previous passes, but I didn't realize some of the fields were out of order (it was effectively a straight copy-paste from ranged.json, aside from damage and adding magazines). Lemme fix that real quick... (or as quick as crappy dialup allows, anyhow).